### PR TITLE
[Cloudflare Logpush] update documentation for s3 and sqs

### PR DESCRIPTION
**Pull request by nathangiuliani**
## What does this PR do?

Update the documentation for configuring the Cloudflare Logpush integration, specifically with SQS.

## Checklist

- [] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

n/a

## How to test this PR locally

n/a

## Related issues

- Relates #5526 

## Screenshots

n/a

### CODE REVIEW

**Comment by efd6** [comment]
@nathangiuliani Please sign the CLA so that we can take a look at this.

**Comment by nathangiuliani** [comment]
> @nathangiuliani Please sign the CLA so that we can take a look at this.

I have.

**Comment by efd6** [comment]
@nathangiuliani Please check that the email address that is associated with the signing is the only git author email address in all commits that are in the PR. If they do not match, please also sign with the newly used email addresses.

**Comment by nathangiuliani** [comment]
> @nathangiuliani Please check that the email address that is associated with the signing is the only git author email address in all commits that are in the PR. If they do not match, please also sign with the newly used email addresses.

Fixed :)

File: packages/cloudflare_logpush/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.6.1"
```
Comment (efd6) [inline packages/cloudflare_logpush/changelog.yml:2]:
```suggestion
- version: "0.7.0"
```
(bump minor for enhancements — also please make the corresponding change in the manifest file)

File: packages/cloudflare_logpush/docs/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ The Cloudflare Logpush integration can be used in three different modes to colle
 
 For example, you could use the data from this integration to know which websites have the highest traffic, which areas have the highest network traffic, or observe mitigation statistics.
 
-## Data streams
+## Logpush Data Sets
```
Comment (efd6) [inline packages/cloudflare_logpush/docs/README.md:14]: Please revert this. The current state is conventional wording.

File: packages/cloudflare_logpush/docs/README.md [hunk 3]
```diff
@@ -40,28 +40,38 @@ This module has been tested against **Cloudflare version v4**.
...
-  | DNS               | dns                    |
-  | Firewall Event    | firewall_event         |
-  | HTTP Request      | http_request           |
-  | NEL Report        | nel_report             |
-  | Network Analytics | network_analytics_logs |
-  | Spectrum Event    | spectrum_event         |
+- Configure [Cloudflare Logpush to Amazon S3](https://developers.cloudflare.com/logs/get-started/enable-destinations/aws-s3/) to send data to an AWS S3 bucket.
+- The default values of the "Bucket Prefix" for each Logpush data set are listed below. However, the user can set the parameter "Bucket Prefix" for each Logpush data set according to their requirements.
+
+  | Logpush Data Set Name | Bucket Prefix          |
```
Comment (efd6) [inline packages/cloudflare_logpush/docs/README.md:46]: Why are these changes needed?

File: packages/cloudflare_logpush/docs/README.md [hunk 4]
```diff
@@ -40,28 +40,38 @@ This module has been tested against **Cloudflare version v4**.
...
-  - For all the event notifications that have been created, select the event type as s3:ObjectCreated:*, select the destination type SQS Queue, and select the queue that has been created in Step 2.
+1. If Logpush forwarding to an AWS S3 Bucket hasn't been configured, then first setup an AWS S3 Bucket as mentioned in the above documentation, and enable forwarding for the relevant account-level or zone-level Logpush data sets.
+2. Follow the steps below for each Logpush data set that has been enabled:
+    1. Create an SQS queue
+        - To setup an SQS queue, follow "Step 1: Create an Amazon SQS queue" mentioned in the [Documentation](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ways-to-add-notification-config-to-bucket.html).
+        - While creating an SQS Queue, please provide the same bucket ARN that has been generated after creating an AWS S3 Bucket.
+    2. Setup event notification from the S3 bucket. Follow this [Link](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications.html). Use the following settings:
```
Comment (efd6) [inline packages/cloudflare_logpush/docs/README.md:62]:
```suggestion
    2. Setup event notification from the S3 bucket according to the documentation [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications.html). Use the following settings:
```

**Review by efd6 — COMMENTED** [review]
Note that the changes in the docs/README.md file should be made in _dev/build/docs/README.md. Then please run `elastic-package build` to render the changes into the docs/README.md file.
